ush/config.sh.RRFS_CONUS_3km_ens

```diff
@@ -126,23 +131,53 @@ if [[ ${DO_ENSEMBLE}  == "TRUE" ]]; then
    DO_ENKF_RADAR_REF="TRUE"
    DO_ENSPOST="TRUE"
    DO_ENSINIT="TRUE"
+
+   RADAR_REF_THINNING="2"
+   ARCHIVEDIR="/5year/BMC/wrfruc/rrfs_ens"
+   CLEAN_OLDFCST_HRS="12"
+   CLEAN_OLDSTMPPOST_HRS="12"
+   cld_bld_hgt=0.0
+   l_precip_clear_only=.true.
+   write_diag_2=.true.
+
+   START_TIME_SPINUP="00:30:00"
+#   LAYOUT_X="11"
+#   LAYOUT_Y="32"
+#   NNODES_RUN_FCST="6"
+
    NUM_ENS_MEMBERS_FCST=9
-   NNODES_RUN_RECENTER="3"
-   PPN_RUN_RECENTER="$(( ${NUM_ENS_MEMBERS} + 1 ))"
    if [[ ${DO_ENSFCST} == "TRUE" ]] ; then
      NUM_ENS_MEMBERS=${NUM_ENS_MEMBERS_FCST}
-#     DO_SPPT=TRUE
-     DO_SPP=TRUE
-#     SPPT_MAG=0.5
      WTIME_RUN_FCST="04:45:00"
+     WTIME_MAKE_LBCS="01:30:00"
+     TAG="cefcst38"
+
+#     LAYOUT_X="31"
+#     LAYOUT_Y="32"
+#     NNODES_RUN_FCST="16"
+
+     DO_SPP="TRUE"
+     DO_SPPT="TRUE"
+     DO_SKEB="FALSE"
+     SPPT_MAG="0.7"
+     DO_LSM_SPP="TRUE"
+
    fi
-   RADAR_REF_THINNING="2"
-   ARCHIVEDIR="/5year/BMC/wrfruc/rrfs_ens"
+   PPN_RUN_RECENTER="$(( ${NUM_ENS_MEMBERS} + 1 ))"
+   NNODES_RUN_RECENTER="3"
```

Comment:
These two lines should be before line 149 so we can have "PPN_RUN_RECENTER" big enough to run the recenter task. If we leave it here, we have PPN_RUN_RECENTER=10 for 9 member ensemble forecast and the recentering task would fail.

Reply:
For ensemble forecast, we have to recenter for 9 members, right?